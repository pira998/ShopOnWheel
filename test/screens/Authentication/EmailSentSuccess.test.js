import React from 'react';
import renderer from 'react-test-renderer';
import EmailSentSuccess from '.../../../screens/Authentication/EmailSentSuccess'
jest.mock('react-native-keyboard-aware-scroll-view', () => {
    const KeyboardAwareScrollView = ({ children }) => children;
    return { KeyboardAwareScrollView };
});

test('renders correctly', () => {
  const tree = renderer.create(<EmailSentSuccess />).toJSON();
  expect(tree).toMatchSnapshot();
});