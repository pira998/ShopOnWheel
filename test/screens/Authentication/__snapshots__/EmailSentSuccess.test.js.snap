// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingVertical": 24,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      }
    }
  />
  <View
    style={
      Object {
        "marginTop": -14,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "Poppins-Bold",
          "fontSize": 22,
          "lineHeight": 30,
          "textAlign": "center",
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#525C67",
          "fontFamily": "Poppins-Regular",
          "fontSize": 16,
          "lineHeight": 22,
          "marginTop": 8,
          "textAlign": "center",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "marginTop": 12,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginBottom": 100,
        }
      }
    >
      <Image
        source={1}
      />
      <Text
        style={
          Object {
            "color": "#53e88b",
            "fontFamily": "Poppins-Bold",
            "fontSize": 30,
          }
        }
      >
        Successfully
      </Text>
      <Text
        style={
          Object {
            "fontFamily": "Poppins-Bold",
            "fontSize": 19,
          }
        }
      >
        Reset Email Sent
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FF6C44",
          "borderRadius": 12,
          "height": 55,
          "justifyContent": "center",
          "marginTop": 24,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontFamily": "sans-serif",
            "fontSize": 16,
            "lineHeight": 22,
          }
        }
      >
        Try Sign in again
      </Text>
    </View>
  </View>
</View>
`;
